docs/start/overview.md

```diff
@@ -85,138 +130,124 @@ And it will generate a TOC for you. Note that there must be at least one content
 file in each folder in order for any sub-folders to be parsed.
 ````
 
-:::{tip}
-Jupyter Book also provides a [cookiecutter](https://github.com/executablebooks/cookiecutter-jupyter-book) that can be used to interactively create a book directory structure.
-The cookiecutter is suitable for more advanced users that want to create a ready-to-go repository to host their book that includes pre-populated metafiles such as README, LICENSE, CONDUCT, CONTRIBUTING, etc., as well as GitHub Actions workflow files to {ref}`publish/gh-actions`.
-To try the cookiecutter template, run the following command:
-
-```bash
-jupyter-book create mybookpath/ --cookiecutter
+```yaml
+# In _toc.yml
+- file: landing-page
+- file: page1.ipynb
```

Comment:
Since the next line states that there should be no extension, should we remove the `.ipynb` here ?

Reply:
ah definitely, will update soon